### DISCUSSION TITLE
http://127.0.0.1:4321/blog/2024-04-20-cdk-notifier-and-tags

### DISCUSSION THREAD

**Discussion by giscus** [JohannesKonings/JohannesKonings.github.io#21] [Blog Post Comments]
# http://127.0.0.1:4321/blog/2024-04-20-cdk-notifier-and-tags

Description of the diff consequence of two ways of tagging in CDK

http://127.0.0.1:4321/blog/2024-04-20-cdk-notifier-and-tags

**Comment by JohannesKonings** [comment]
test
